internal/migrations/015-system-roles-bootstrap.sql

```diff
@@ -22,4 +22,8 @@ ON CONFLICT (owner, role_name) DO NOTHING;
 -- Set the network_writer role to be managed by the network_writers_manager role
 UPDATE roles 
 SET manager_owner = 'system', manager_role_name = 'network_writers_manager' 
-WHERE owner = 'system' AND role_name = 'network_writer'; 
\ No newline at end of file
+WHERE owner = 'system' AND role_name = 'network_writer';
+
+INSERT INTO role_members (owner, role_name, wallet, granted_at, granted_by) VALUES
```

Comment:
I don't believe it's good to hardcode this (i.e. 0xb96c7b062117d9b0361cb69358dfcff00f857f9e)

I mean, it's more maintainable to have a script that executes the insert exec sql with a variable for the address, if we want to automate it